frontend/src/api/contexts/config/types.ts

```diff
@@ -108,6 +108,7 @@ export interface Home {
   youtubeVideoId: string
   showEvents: boolean
   showNews: boolean
+  images: string[]
```

Comment:
There is no such field in HomeComponent, but there is an api endpoint for retrieving images from gallery: /api/gallery

Reply:
But I created an endpoint for it here
https://github.com/kir-dev/cmsch/pull/764

Reply:
I merged it, you can rebase to use it